pkg/registry/apis/provisioning/register.go

```diff
@@ -128,8 +129,14 @@ func NewAPIBuilder(
 	extraBuilders []ExtraBuilder,
 	extraWorkers []jobs.Worker,
 	jobHistoryConfig *JobHistoryConfig,
+	newClientFactoryOnlyApiServer func(apiserver.RestConfigProvider) resources.ClientFactory, // optional, only used when onlyApiServer is true
```

Comment:
@stephaniehingtgen any recs on what to name this? Seems onlyApiServer isn't what it was. I was using it as `isStandalone`. Is it possible to add `isStandalone` to describe intent, or given this is the only param that is standalone specific, for now, should just rename this to `newStandaloneClientFactoryFunc`?